hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1753,17 +1770,26 @@ private void checkAcls(ResourceType resType, StoreType store,
         .setClientUgi(user)
         .setIp(ProtobufRpcEngine.Server.getRemoteIp())
         .setAclType(ACLIdentityType.USER)
-        .setAclRights(acl)
+        .setAclRights(aclType)
         .build();
     if (!accessAuthorizer.checkAccess(obj, context)) {
       LOG.warn("User {} doesn't have {} permission to access {}",
-          user.getUserName(), acl, resType);
-      throw new OzoneAclException("User " + user.getUserName() + " doesn't " +
-          "have " + acl + " permission to access " + resType,
-          ErrorCode.PERMISSION_DENIED);
+          user.getUserName(), aclType, resType);
```

Comment:
The unit test failure seems related:
TestOmAcls.testOMAclsPermissionDenied
org.apache.hadoop.ozone.om.exceptions.OMException: User jenkins1000 doesn't have CREATE permission to access volume

Reply:
Yes @xiaoyuyao. Will address in the next patch along with review comments.

Reply:
Fixed.